test/JDBC/expected/cast_nvarchar_test-vu-prepare.out

```diff
@@ -1,14 +1,13 @@
+CREATE TABLE TestHash(
```

Comment:
can we have some tests where we create userdefined hashbytes and create view/func/proc which is dependent on that?